homeassistant/components/fyta/strings.json

```diff
@@ -36,51 +36,54 @@
       "plant_status": {
         "name": "Plant state",
         "state": {
-          "too_low": "Too low",
-          "low": "Low",
-          "perfect": "Perfect",
-          "high": "High",
-          "too_high": "Too high"
+          "deleted": "Deleted",
+          "doing_great": "Doing great",
+          "need_attention": "Needs attention",
+          "no_sensor": "No sensor"
         }
       },
       "temperature_status": {
         "name": "Temperature state",
         "state": {
-          "too_low": "[%key:component::fyta::entity::sensor::plant_status::state::too_low%]",
-          "low": "[%key:component::fyta::entity::sensor::plant_status::state::low%]",
-          "perfect": "[%key:component::fyta::entity::sensor::plant_status::state::perfect%]",
-          "high": "[%key:component::fyta::entity::sensor::plant_status::state::high%]",
-          "too_high": "[%key:component::fyta::entity::sensor::plant_status::state::too_high%]"
+          "no_data": "No data",
```

Comment:
Should we return None when this happens? It would render as Unknown, which sounds about the same.

Reply:
In my view unknown should be shown, if HA does not know the value/state. However, if the API reports no_data, this means that FYTA has not (yet) sufficient data to calculate the state. This is in particular during the setup, where it takes up to one day until a proper state is reported. Same could apply, if sensor is recalibrated.

Reply:
it sounds kinda strange imo to have Unknown, Unavailable and No data - 3 states to tell the users we have no data. But your reasoning makes sense. Do we have a way to tell if its calibrating?